### PR TITLE
UCT/API: Update UCT iface tag caps

### PR DESCRIPTION
**Pull request by alex-mikheev**
@yosefe 

Add max_zcopy for receive operation
Add max_outstanding for receive operations.

### CODE REVIEW

File: src/uct/ib/rc/verbs/rc_verbs_iface.c [hunk 1]
```diff
@@ -681,11 +681,13 @@ void uct_rc_verbs_iface_tag_query(uct_rc_verbs_iface_t *iface,
 
         iface_attr->cap.tag.eager.max_bcopy = ib_iface->config.seg_size - eager_hdr_size;
         iface_attr->cap.tag.eager.max_zcopy = ib_iface->config.seg_size - eager_hdr_size;
+        iface_attr->cap.tag.eager.max_iov   = 1;
+
```
Comment (shamisp) [inline src/uct/ib/rc/verbs/rc_verbs_iface.c]: what about updating all the rest ?

Comment (yosefe) [inline src/uct/ib/rc/verbs/rc_verbs_iface.c]: transports which do not have tag matching support do not have to set this value

**Comment by yosefe** [comment]
@bbenton pls take a look

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4275/ for details (Mellanox internal link).

File: src/uct/ib/rc/verbs/rc_verbs_iface.c (hunk 1 above)
Comment (shamisp) [inline src/uct/ib/rc/verbs/rc_verbs_iface.c]: oh... missed this

**Comment by shamisp** [comment]
👍

**Comment by bbenton** [comment]
:+1: for doc/api

**Pull request merged**